hexrd/ui/main_window.py

```diff
@@ -285,6 +285,11 @@ def open_image_files(self):
 
             if dialog.exec_():
                 detector_names, image_files = dialog.results()
+                image_files = [img for f in files for img in f]
+                files = [[] for det in HexrdConfig().detector_names]
```

Comment:
Just for interest you can achieve the same with:

```python
[[]] * len(HexrdConfig().detector_names)
```
No  need to change, just added for interest :-)